.aicoe-ci.yaml

```diff
@@ -0,0 +1,17 @@
+check:
+  - thoth-precommit
+  - thoth-build
+build:
+  base-image: "registry.access.redhat.com/ubi8"
```

Comment:
Is the base image necessary in docker file way to build?

Reply:
No, it is not.please remove it.

Reply:
Thanks